dotnet/src/Agents/Abstractions/ChatHistoryKernelAgent.cs

```diff
@@ -17,13 +18,14 @@ protected internal sealed override IEnumerable<string> GetChannelKeys()
     }
 
     /// <inheritdoc/>
-    protected internal sealed override Task<AgentChannel> CreateChannelAsync(CancellationToken cancellationToken)
+    protected internal sealed override Task<AgentChannel> CreateChannelAsync(ILogger logger, CancellationToken cancellationToken)
     {
         return Task.FromResult<AgentChannel>(new ChatHistoryChannel());
     }
```

Comment:
`ILogger` is added here and it's not used in a method. So, the idea of adding `ILogger` on class level should resolve this issue.

Reply:
Nothing interesting happening here.  Agents don't have much interesting so say / log.  (See previous comments).

Here where it is used:

https://github.com/microsoft/semantic-kernel/blob/feature_agent_logging/dotnet/src/Agents/OpenAI/OpenAIAssistantAgent.cs#L207